test/js/main.js

```diff
@@ -0,0 +1,191 @@
+//require("@babel/register");
+const {Builder} = require('selenium-webdriver');
+const remote = require('selenium-webdriver/remote');
+const config = require('./config.json');
+const Mocha = require('mocha');
+import {createBrowserStackLocal,stopBrowserstackLocal} from './utils/browserstack'
+import {eachP,asyncForEach} from './utils/async'
+import {spawnP} from './utils/misc'
+import {exec} from 'child_process'
+
+// Input capabilities
+const bsCapabilitiesDefault = {
+  'acceptSslCerts' : 'true',
+  'browserstack.debug': "true",
+   project: 'JS SDK',
+  'browserstack.user' : process.env.BROWSERSTACK_USERNAME,
+  'browserstack.key' : process.env.BROWSERSTACK_ACCESS_KEY,
+  'browserstack.local' : 'true',
+  "unexpectedAlertBehaviour": "dismiss",
```

Comment:
[Minor] Yet again inconsistency between single and double quotes - IMO ESLint should take care of that.

Reply:
yep, agreed